feature/landing-security (#2575)
mercadopago/devsite-docs: guides/additional-content/security/landing-security.en.md

# Security

At Mercado Pago, we have implemented a series of security measures designed to protect customer and user payments, ensure confidentiality and integrity in all processes, and provide greater security in the integrations that our payment solutions offer.

> Below, we outline the protocols used by Mercado Pago. Click on **Learn more** to find more information.

Author: ```suggestion
Below, we outline the protocols used by Mercado Pago.
```

## OAuth

OAuth (Open Authorization) is an authorization protocol that allows applications to gain limited access to user accounts on an HTTP service, such as social networks, without the user having to share their credentials. Instead, OAuth defines a method for users to grant third-party applications access to their data without needing to reveal their login information.

> [Learn more](/developers/en/docs/security/oauth/introduction)

Author: ```suggestion
For more information, access the [documentation](/developers/en/docs/security/oauth/introduction).
```

## OWASP

Open Web Application Security Project (OWASP) is an open and secure community that provides tools and standards for the development and maintenance of web applications. It aims to promote the research and development of security in applications. Through its initiatives, OWASP contributes to raising the security standard in the software industry and creating a safer online community.

> [Learn more](/developers/es/docs/security/owasp)

Author: ```suggestion
For more information, access the [documentation](/developers/es/docs/security/owasp).
```

## PCI DSS

PCI DSS (Payment Card Industry Data Security Standard) is an international security standard that all entities storing, processing, or transmitting card data must comply with. It is one of the most demanding security standards in the payment industry, which Mercado Pago adheres to, allowing it to operate with credit and debit cards.

> [Learn more](/developers/en/docs/security/pci)

Author: ```suggestion
For more information, access the [documentation](/developers/en/docs/security/pci).
```
